appengine-java8/search/src/main/java/com/example/appengine/search/SearchServlet.java

```diff
@@ -20,32 +18,31 @@
 // [START search_document_import]
 import com.google.appengine.api.search.Document;
 import com.google.appengine.api.search.Field;
+import com.google.appengine.api.search.Index;
+import com.google.appengine.api.search.IndexSpec;
 import com.google.appengine.api.search.Results;
 import com.google.appengine.api.search.ScoredDocument;
 import com.google.appengine.api.search.SearchException;
-import com.google.appengine.api.search.StatusCode;
-// [END search_document_import]
-
-// CHECKSTYLE:OFF
-import com.google.appengine.api.search.Index;
-import com.google.appengine.api.search.IndexSpec;
 import com.google.appengine.api.search.SearchServiceFactory;
-// @formatter:on
-// CHECKSTYLE:ON
-
+import com.google.appengine.api.search.StatusCode;
 import java.io.IOException;
 import java.io.PrintWriter;
-
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-
+// [END search_document_import]
```

Comment:
❗️ These imports seem also to have been expanded beyond that which was intended.